src/index.js

```diff
@@ -214,12 +230,35 @@ export default class RNPickerSelect extends PureComponent {
         return {};
     }
 
+    async measureTextInput() {
+        return new Promise((resolve) => {
+            this.inputRef.measureInWindow((_x, y, _width, height) => {
+                resolve({ y, height });
+            });
+        });
+    }
+
+    async handleScrollOnOpenClose() {
```

Comment:
```suggestion
    async scrollToInput() {
```
We prefer naming methods for what they do and not what they handle (makes them easier to reuse and understand the intention).